githubkit/auth/app.py

```diff
@@ -287,8 +287,8 @@ def as_installation(
             installation_id,
             self.client_id,
             self.client_secret,
-            repositories,
-            repository_ids,
+            UNSET if repositories is UNSET else list(repositories),
```

Comment:
This will make a copy of the data anyway. I think it's better to only allow `Sequence` here. Users are responsible for making the sequence.

Reply:
I had started with all `repository_ids` arguments as iterables, before seeing that they need to be iterated over more than once. After accounting for that I didn't quite realise this was the only place left still using `Iterable`. I'll downgrade this to `Sequence`.